package.nls.de.json

```diff
@@ -1,23 +1,23 @@
 {
-  "command.regenerateIcons.title": "Icon-Veränderungen anwenden",
+  "command.regenerateIcons.title": "Icon-Anpassungen anwenden",
   "command.ngPreset.title": "Angular Preset umschalten (Workspace Level)",
   "command.jsPreset.title": "Offizielles JS Preset umschalten (User Level)",
   "command.tsPreset.title": "Offizielles TS Preset umschalten (User Level)",
   "command.jsonPreset.title": "Offizielles JSON Preset umschalten (User Level)",
   "command.hideFoldersPreset.title": "Ordner-Icons Sichtbarkeit umschalten (User Level)",
   "command.foldersAllDefaultIconPreset.title": "Spezifische Order-Icons umschalten (User Level)",
   "command.restoreIcons.title": "Icon-Manifest zurücksetzen",
-  "command.resetProjectDetectionDefaults.title": "Projekterkennung zurücksetzen",
-  "configuration.title": "vscode-icons Einstellungen",
+  "command.resetProjectDetectionDefaults.title": "Bereits erkannte Projekte zurücksetzen",
```

Comment:
@jens1o, @JimiC I, I thing using the form that the `vscode  team` is using would be maybe the way to go. 

In the case of the Spanish translations I did them using the informal form but then I saw that they were using the formal form so... what do you think?